hydra autoMigration pod crashing

**Describe the bug**

Deploying `ory/hydra` using `ory/k8s` charts if `autoMigration` is set to `true` the pod will crash and backoff with the error below. 

This is due to missing `--config` is given. A fix will be passing `--config` argument to the initContainer. Instead of fixing it at `rootCmd` of `hydra`.

```
Config file not found because "Config File ".hydra" Not Found in "[/]""
panic: runtime error: invalid memory address or nil pointer dereference
[signal SIGSEGV: segmentation violation code=0x1 addr=0xd8 pc=0xac53f9]

goroutine 1 [running]:
github.com/ory/x/urlx.ParseOrFatal(0x0, 0x0, 0xc000038004, 0x75, 0x0)
	/go/pkg/mod/github.com/ory/x@v0.0.64/urlx/parse.go:22 +0x69
github.com/ory/hydra/driver.(*RegistrySQL).CanHandle(0xc00040f710, 0xc000038004, 0x75, 0x0)
	/go/src/github.com/ory/hydra/driver/registry_sql.go:160 +0x54
github.com/ory/x/dbal.GetDriverFor(0xc000038004, 0x75, 0x75, 0x174e740, 0x7fbc8c5236d0, 0x0)
	/go/pkg/mod/github.com/ory/x@v0.0.64/dbal/driver.go:36 +0x85
github.com/ory/hydra/driver.NewRegistry(0x105dec0, 0xc000076d20, 0x0, 0x0, 0x0, 0xc000076b40)
	/go/src/github.com/ory/hydra/driver/registry.go:59 +0x61
github.com/ory/hydra/driver.NewDefaultDriver(0x1058580, 0xc000076b40, 0x0, 0x0, 0x0, 0x0, 0x0, 0x0, 0x0, 0x0, ...)
	/go/src/github.com/ory/hydra/driver/driver_default.go:21 +0xf6
github.com/ory/hydra/cmd/cli.(*MigrateHandler).MigrateSQL(0x1770720, 0x173b7c0, 0xc0004368c0, 0x0, 0x2)
	/go/src/github.com/ory/hydra/cmd/cli/handler_migrate.go:50 +0x9ff
github.com/spf13/cobra.(*Command).execute(0x173b7c0, 0xc000436860, 0x2, 0x2, 0x173b7c0, 0xc000436860)
	/go/pkg/mod/github.com/spf13/cobra@v0.0.3/command.go:766 +0x2ae
github.com/spf13/cobra.(*Command).ExecuteC(0x173ba20, 0xdba640, 0x1770720, 0x0)
	/go/pkg/mod/github.com/spf13/cobra@v0.0.3/command.go:852 +0x2ec
github.com/spf13/cobra.(*Command).Execute(...)
	/go/pkg/mod/github.com/spf13/cobra@v0.0.3/command.go:800
github.com/ory/hydra/cmd.Execute()
	/go/src/github.com/ory/hydra/cmd/root.go:59 +0x32
main.main()
	/go/src/github.com/ory/hydra/main.go:33 +0x50
```

**To Reproduce**

Steps to reproduce the behavior:

1. Deploy a brand new hydra using `ory/hydra` helm chart with `autoMigration: true`
2. `hydra` pod will never be ready due to `initContainer` for migration crashing.

**Expected behavior**

Migration should happen and eventually `hydra` pod will be ready.

**Environment**

* Version: v1.2.3, git sha hash
* Environment: Debian, Docker, ...

**Additional context**

Add any other context about the problem here.

--- a/helm/charts/hydra/templates/deployment.yaml
+++ b/helm/charts/hydra/templates/deployment.yaml
@@ -31,7 +31,11 @@ spec:
           image: "{{ .Values.image.repository }}:{{ .Values.image.tag }}"
           imagePullPolicy: {{ .Values.image.pullPolicy }}
           command: ["hydra"]
-          args: ["migrate", "sql", "-e", "--yes"]
+          args: ["migrate", "sql", "-e", "--yes", "--config", "/etc/config/config.yaml"]
+          volumeMounts:
+            - name: {{ include "hydra.name" . }}-config-volume
+              mountPath: /etc/config
+              readOnly: true
           env:
             - name: DSN
               valueFrom:

